$MirOS: ports/lang/python/2.3/patches/patch-setup_py,v 1.3 2007/07/19 20:26:15 bsiegert Exp $
$OpenBSD: patch-setup_py,v 1.1.1.1 2003/12/31 17:38:33 sturm Exp $
--- setup.py.orig	2008-02-05 00:41:02.000000000 +0100
+++ setup.py	2008-04-22 23:12:11.000000000 +0200
@@ -911,7 +911,7 @@ class PyBuildExt(build_ext):
                 exts.append( Extension('resource', ['resource.c']) )
 
             # Sun yellow pages. Some systems have the functions in libc.
-            if platform not in ['cygwin', 'atheos']:
+            if platform not in ['cygwin', 'atheos'] and platform[0:6] != 'mirbsd':
                 if (self.compiler.find_library_file(lib_dirs, 'nsl')):
                     libs = ['nsl']
                 else:
@@ -1535,8 +1535,7 @@ def main():
           ext_modules=[Extension('_struct', ['_struct.c'])],
 
           # Scripts to install
-          scripts = ['Tools/scripts/pydoc', 'Tools/scripts/idle',
-                     'Lib/smtpd.py']
+          scripts = ['Lib/smtpd.py']
         )
 
 # --install-platlib
